packages/contracts-bedrock/snapshots/storageLayout/L1StandardBridge.json

```diff
@@ -36,24 +22,45 @@
   },
   {
     "bytes": "20",
-    "label": "messenger",
+    "label": "spacer_3_0_20",
     "offset": 0,
     "slot": "3",
-    "type": "contract ICrossDomainMessenger"
```

Comment:
moved to the end to account for the L2StandardBridge which now uses a constant for this value.